release-notes/WIP-Cabal-3.12.x.0.md

```diff
@@ -0,0 +1,19 @@
+Pre-release Cabal 3.12.1.0 changelog and release notes.
```

Comment:
Thanks a lot for the release note. Could you remove this first line? It made sense for the pre-release of cabal-install 3.12, but we are/were not pre-releasing Cabal the library.